package.json

```diff
@@ -15,12 +15,11 @@
     "cors": "^2.8.5",
     "dotenv": "^8.2.0",
     "express": "^4.17.1",
-    "knex": "^0.21.5",
-    "pg": "^7.18.2"
-  },
-  "devDependencies": {
+    "knex": "^0.21.1",
+    "pg": "^8.0.3",
     "@babel/node": "^7.10.5",
     "@babel/plugin-proposal-class-properties": "^7.10.4",
     "@babel/preset-env": "^7.11.5"
-  }
+  },
+  "devDependencies": {}
```

Comment:
What's this extra empty object all about?

Reply:
good catch. It's not doing anything. I moved @bable/node, plugin-proposal-etc, and preset-env into dependencies. They probably can move back, since some endpoints were responding just fine where they were. 

The penultimate issue was the helper functions couldn't access the database, and endpoints that didn't use helpers worked just fine. So other than those changes and the migrations, likely most of this can go back to the way it was in V1.